src/mg/PAMI/Logger/Log4PhpAdapter.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace PAMI\Logger;
+
+use Logger;
+use LoggerLevel as Log4PhpLevel;
+use LoggerLevel;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel as PsrLevel;
+
+class Log4PhpAdapter implements LoggerInterface
```

Comment:
@jacobkiers I was just thinking.. would you like to have this adapter as a separate project and include it as a dependency? Or maybe even better: open a pull request to the log4php guys. I just thought about this because we would need to duplicate this code for pagi